docker-compose.yml

```diff
@@ -308,6 +308,7 @@ services:
         - DBUSER=${DBUSER}
         - DBPASS=${DBPASS}
         - SKIP_LETS_ENCRYPT=${SKIP_LETS_ENCRYPT:-n}
+        - SKIP_ECDSA_CERT=${SKIP_ECDSA_CERT:-n}
```

Comment:
This has to default to `y` since not everybody uses update.sh.

Reply:
You're right.